assets/js/apps/tiles/components/DatasetFilter.jsx

```diff
@@ -18,81 +18,70 @@ const DatasetFilter = ({ onFilter, onSort }) => {
   };
   return (
     <Fragment>
-      <p>
-        <div className="row">
-          <div className="form-horizontal col-md-8">
-            <div className="form-group has-search">
-              {/* <span className="fa fa-search form-control-feedback"/> */}
-              <input
-                type="text"
-                className="form-control"
-                placeholder="Just start typing to filter datasets based on description"
-                onKeyUp={onFilter}
-              />
-            </div>
+      <div className="row mb-3">
+        <div className="col-md-8">
+          <div className="has-search">
+            <input
+              type="text"
+              className="form-control"
+              placeholder="Start typing to filter datasets based on description"
+              onKeyUp={onFilter}
+            />
           </div>
-          <div className="form-horizontal col-md-4">
-            <div className="btn-group">
-              <button type="button" className="btn btn-outline-secondary float-right">Sort by</button>
-              <button
-                type="button"
-                className="btn btn-outline-secondary dropdown-toggle dropdown-toggle-split"
-                data-toggle="dropdown"
-                aria-haspopup="true"
-                aria-expanded="false"
-              >
-                <span className="sr-only">Toggle Dropdown</span>
-              </button>
-              <div className="dropdown-menu" style={dropDOwnStyle} ref={(node) => { if (node) { node.style.setProperty('transform', 'translate3d(0px 38px 0px)', '!important'); } }}>
-                <div className="row">
-                  <div className="col col-md-8">
-                    <span className="dropdown-item" href="#">
-                      Description
-                    </span>
-                  </div>
-                  <div className="col col-md-4">
-                    <button className="fa fa-sort-alpha-asc mr-2 mt-2" style={sortButtonStyle} type="button" onClick={() => onSort('description', 'asc')} />
-                    <button className="fa fa-sort-alpha-desc mt-2" type="button" style={sortButtonStyle} onClick={() => onSort('description', 'desc')} />
-                  </div>
+        </div>
+        <div className="col-md-4">
+          <div className="btn-group">
+            <button type="button" className="btn btn-outline-secondary float-end">Sort by</button>
+            <button
+              type="button"
+              className="btn btn-outline-secondary dropdown-toggle dropdown-toggle-split"
+              data-bs-toggle="dropdown"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 28 spaces but found 14.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
                            data-bs-toggle="dropdown"
```